.pre-commit-config.yaml

```diff
@@ -0,0 +1,27 @@
+repos:
+- repo: https://github.com/ambv/black
+  rev: 18.9b0
+  hooks:
+  - id: black
+    language_version: python3
+- repo: https://github.com/asottile/blacken-docs
+  rev: v0.3.0
+  hooks:
+  - id: blacken-docs
+    additional_dependencies: [black==18.9b0]
+- repo: https://github.com/asottile/pyupgrade
+  rev: v1.11.1
+  hooks:
+  - id: pyupgrade
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.1.0
+  hooks:
+  - id: trailing-whitespace
+  - id: end-of-file-fixer
+  - id: check-yaml
+  - id: check-ast
+  - id: mixed-line-ending
+    args: ["--fix=lf"]
+  - id: debug-statements
+  - id: flake8
```

Comment:
flake8 official repo now exports its own pre-commit hook, e.g. see https://github.com/asottile/pyupgrade/pull/95

Reply:
Oh nice. Can't find an immediate mention of bugbear but it doesn't seem to complain about the source.